cosimulation/icarus/test/Makefile

```diff
@@ -0,0 +1,2 @@
+all:
+	python test_all.py
```

Comment:
why have a separate directory for icarus?  Do these tests not fit the structure of the structure of the tests in general or toVerilog?